docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -1360,7 +1360,7 @@ export interface SdkModelType extends SdkTypeBase {
   generatedName: string;
   description?: string;
   details?: string;
-  access?: AccessFlags;
+  access: AccessFlags;
```

Comment:
@archerzz Our goal is to take the types from tcgc and directly serialize them into codemodle.json file.  Right now the decorator representation work is not done so you won't be able to serialize this as is and get the information on the other side when you deserialize unless you synthetically add a property `wasAccessOverriden`.  What are your thoughts on doing that in the short term until the decorator work is complete vs having this pr wait for the decorator work to happen first and then we do this?

Reply:
> @archerzz Our goal is to take the types from tcgc and directly serialize them into codemodle.json file. Right now the decorator representation work is not done so you won't be able to serialize this as is and get the information on the other side when you deserialize unless you synthetically add a property `wasAccessOverriden`. What are your thoughts on doing that in the short term until the decorator work is complete vs having this pr wait for the decorator work to happen first and then we do this?

@m-nash I think we can go with a workaround. We can use `__raw` to check if the `@access` is explicitly set.

Reply:
I think it's fair to use `.__raw` for now, will wait on chat discussion to see what happens

Reply:
@archerzz can you link the follow up issue to stop using __raw and start using .decorators once that is in place so we don't lose track of that?